docs/tutorials/mpv.md

```diff
@@ -1,144 +1,139 @@
 ---
-label: MPV
-description: Installation, setup, and configuration for MPV
+label: mpv
+description: Learn how to install, setup, and configure mpv
 image: https://user-images.githubusercontent.com/78981416/215125796-08b99128-fe50-4d0c-b0dd-49f8828af0dc.png
 ---
 
-# MPV
+# mpv
 
-[MPV](https://mpv.io/) is a versatile and lightweight media player with extensive customization options. It supports a wide variety of video file formats, audio and video codecs, subtitle types, offers advanced features, and is compatible with multiple platforms.
+[mpv](https://mpv.io) is a versatile and lightweight media player with extensive customizability. It is used and recommended by most users as it supports a wide range of video file formats, audio and video codecs, and subtitle types. Additionally, it offers many advanced features and is available on multiple platforms.
 
 ## Installation
 
 ### Windows
 
-==- Manually Installing MPV
+==- Installing the Source Binary
```

Comment:
The average person will not know what a "source binary" is due it being a relatively uncommon word for the non tech-savvy ones